suite-native/intl/src/__tests__/localeSlice.test.ts

```diff
@@ -0,0 +1,53 @@
+import { LocaleSliceRootState, selectIsLanguageLocalizationSupported } from '../localeSlice';
+
+describe('selectIsLanguageLocalizationSupported', () => {
```

Comment:
Why not use parametrized test for this? A lot of boilerplate while the testing logic is the same.